bin/ream/src/main.rs

```diff
@@ -56,11 +57,26 @@ async fn main() {
                 subnets: Subnets::new(),
             };
 
-            let ream_db = ReamDB::new(config.data_dir, config.ephemeral)
+            // database must be clean for checkpoint sync
+            if config.purge_db {
+                reset_db(config.data_dir.clone()).expect("Unable to delete database");
+            }
+
+            let ream_db = ReamDB::new(config.data_dir.clone(), config.ephemeral)
                 .expect("unable to init Ream Database");
 
             info!("ream database initialized ");
 
+            let _store;
+            let _slot;
+            (_store, _slot) =
+                initialize_db_from_checkpoint(ream_db.clone(), config.checkpoint_sync_url)
+                    .await
+                    .map_err(|err| panic!("{err}"))
+                    .unwrap();
```

Comment:
```suggestion
           initialize_db_from_checkpoint(ream_db.clone(), config.checkpoint_sync_url)
                    .await
                    .map_err(|err| panic!("{err}"))
                    .unwrap();
```

Why are we returning store? if we already have access to it, also slot isn't used so why return it?